### PR TITLE
#24 client login page

### PR DESCRIPTION
**Pull request by cedrickmandocdoc**
Checkout from #25 #22

### CODE REVIEW

File: src/client/app/public/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      email: ['', Validators.required],
+      password: ['', Validators.required]
+    });
+  }
+
+  public onClickLogin() {
+    this.login(this.loginForm.getRawValue());
+  }
+
+  private login(data: {email: string, password: string}) {
```
Comment (jmaicaaan) [inline src/client/app/public/login/login.component.ts]: This is getting a little bit dirty to look. Can we have the reference here to the entity?

**Comment by jmaicaaan** [comment]
Also please fix merge conflicts. Thank you

**Comment by cedrickmandocdoc** [comment]
I've removed here the pug loader in package, because it is not on the webpack config on the first place. If we are using pug and styl. Please add another issue for this. And label as a blocker.

File: src/client/app/public/login/login.component.ts (hunk 1 above)
Comment (jmaicaaan) [reply src/client/app/public/login/login.component.ts]: With that, I think it is better to remove the data. Only email and password.

Comment (cedrickmandocdoc) [reply src/client/app/public/login/login.component.ts]: Yeah we can do that, I just want to accept an object if the data is more than one and related to each other. I think it should be an object, for unity to other functions and not to populate the parameters of a function.

Comment (jmaicaaan) [reply src/client/app/public/login/login.component.ts]: That's reasonable enough 👍

**Pull request merged**